platform-ajax/src/main/java/com/softicar/platform/ajax/request/AjaxRequestMessageFetcher.java

```diff
@@ -0,0 +1,59 @@
+package com.softicar.platform.ajax.request;
+
+import com.softicar.platform.common.core.exceptions.SofticarIOException;
+import com.softicar.platform.common.io.HexDecoderStream;
+import com.softicar.platform.common.io.StreamUtils;
+import com.softicar.platform.common.string.charset.Charsets;
+import java.io.IOException;
+import java.util.Optional;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+
+class AjaxRequestMessageFetcher {
+
+	private static final String FORM_REQUEST_INPUT_NAME = "form-request-input";
+	private final HttpServletRequest request;
+	private final String contentType;
+
+	AjaxRequestMessageFetcher(HttpServletRequest request) {
+
+		this.request = request;
+		this.contentType = request.getContentType();
+	}
+
+	Optional<AjaxRequestMessage> fetchMessage() {
+
+		if (isContentType("application/json")) {
```

Comment:
We could use `com.softicar.platform.common.io.mime.MimeType` here

Reply:
did so